### ISSUE TITLE
GRPC Client doesn't accept `repeated float` from Python Service

### ISSUE THREAD

**Issue by rukmal** [ballerina-platform/ballerina-library#5080]
**Description:**
Tl; dr: I have a GRPC service specification that I used to generate a Python service with. The Python service returns a float `[1.0, 2.0]`. However, in the generated Ballerina client, when I print out the response, I see `[]`.

Curiously, when I generate a server with Ballerina, and repeat the same test, it works without issue. The Python GRPC service works as expected when consumed with Postmant.

Full report with reproduction steps here: https://discord.com/channels/957996897782616114/1166106171959152640/1166106171959152640
**Steps to reproduce:**

See zip file for reproduction source code (Ballerina GRPC Client, Python GRPC Service, Ballerina GRPC Service).

**Affected Versions:**

Tested 2201.7.* and 2201.8.*

**OS, DB, other environment details and versions:**

MacOS Ventura 15.4

**Related Issues (optional):**

**Suggested Labels (optional):**

grpc

**Suggested Assignees (optional):**

**Comment by rukmal** [comment]
+1 on this - is there a workaround in the meantime?

**Comment by dilanSachi** [comment]
Closing as this is fixed
